app/posts/publish-teacher-training-courses/2018-08-07-success-messaging.md

```diff
@@ -2,15 +2,17 @@
 title: Success messaging
 description: Messages we show after a successful save or publish.
 date: 2018-08-07
+tags:
+ - PN001
```

Comment:
Why PN and not PU like the existing need, PU001?

Reply:
I renamed it to PN to match the naming convention in the other sections. e.g. MN, AN... PN.